example/example/tests/conftest.py

```diff
@@ -35,4 +35,7 @@ def _create_env_file(**kwargs):
 @pytest.fixture
 def astrosat_settings():
     astrosat_settings = AstrosatSettings.load()
+    # make sure logging is enabled...
+    astrosat_settings.enable_db_logging = True
+    astrosat_settings.save()
```

Comment:
I wouldn't hard-code `enable_db_logging` to True in this fixture.  Because other tests that don't necessarily want to do that may also use this fixture.  Instead, move lines 39 & 40 into whichever tests use this fixture _and_ need to enable db logging.

Reply:
The bits of code below that you previously had but deleted would work just fine.

Reply:
I initially did this as this fixture already existed, but did it to save code duplication, but your right, that was a bad idea. I'll revert it back to what I had before.